### DISCUSSION TITLE
0.13.3

### DISCUSSION THREAD

**Discussion by SchrodingersGat** [inventree/InvenTree#6312] [Announcements]
### Bug Fixes
* [0.13.x] Do not create owners when importing data (#6271) by @github-actions in https://github.com/inventree/InvenTree/pull/6272
* Import Fix (#6274) by @SchrodingersGat in https://github.com/inventree/InvenTree/pull/6277
* JS translation fix (#6288) by @SchrodingersGat in https://github.com/inventree/InvenTree/pull/6289
* [0.13.x] django-allauth==0.6.1 (#6301) by @github-actions in https://github.com/inventree/InvenTree/pull/6302
* SSO Error Improvememts (#6246) by @SchrodingersGat in https://github.com/inventree/InvenTree/pull/6304
### Other Changes
* Update version.py by @SchrodingersGat in https://github.com/inventree/InvenTree/pull/6266

**Full Changelog**: https://github.com/inventree/InvenTree/compare/0.13.2...0.13.3

<hr /><em>This discussion was created from the release <a href='https://github.com/inventree/InvenTree/releases/tag/0.13.3'>0.13.3</a>.</em>

**Comment by mittkkoo** [comment]
Great work!
How are releases uploaded on packager.io? Currently the stable branch is only at 13.0.

**Reply by matmair** [reply]
Thanks for the report, seems like the automation did not trigger

**Reply by matmair** [reply]
@mittkkoo 0.13.3 just finished building, it should be visible in the mirrors in a few seconds
